.pre-commit-config.yaml

```diff
@@ -10,15 +10,10 @@ repos:
     rev: 22.8.0
     hooks:
     -   id: black
--   repo: http://github.com/PyCQA/isort
-    rev: 5.10.1
+-   repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.137
```

Comment:
we have ruff>=0.0.138,<0.1 in `test-requirements`

Reply:
Yeah but there's no `v0.0.138` tag on the pre-commit repo yet :confused:

Reply:
I'd rather use the local ruff version
```
  - id: lint
    name: lint
    entry: make lint
    types: [python]
    language: system
    pass_filenames: false
```

Reply:
Ah sweet, didn't know this was possible with pre-commit :+1: Will update!

Reply:
@PrettyWood Done, for both python and rust. For now it's fast enough, so it runs on everything, anytime a rust or python file changes. If it gets slow in the future, I'll split the `make` targets per language